python/sparkdl/utils/graph_utils.py

```diff
@@ -0,0 +1,163 @@
+#
```

Comment:
why not put it in python/sparkdl/graph/utils.py ? I think it's better to put specialized util modules in the relevant submodules vs putting all util modules in the same "utils" module.